SS_write_ssnew.tpl

```diff
@@ -1556,75 +1586,29 @@ FUNCTION void write_nucontrol()
     }
   }
 
-  NuFore << Fcast_Loop_Control(1) << " #_N forecast loops (1=OFL only; 2=ABC; 3=get F from forecast ABC catch with allocations applied)" << endl;
-  NuFore << Fcast_Loop_Control(2) << " #_First forecast loop with stochastic recruitment" << endl;
-  NuFore << Fcast_Loop_Control(3) << " #_Forecast recruitment:  0= spawn_recr; 1=mult*spawn_recr_fxn; 2=mult*VirginRecr; 3=recent mean for recr and recr_dist from yr range above; 4=mult * recent mean for recr only" << endl;
-  NuFore << "#_ for options 3 and 4, need to set phase for fore_recr_devs to -1 in control to get requested mean in MCMC" << endl;
+  NuFore << "#" << endl << Fcast_Loop_Control(1) << " #_N forecast loops (1=OFL only; 2=ABC; 3=get F from forecast ABC catch with allocations applied)" << endl;
+  NuFore << Fcast_Loop_Control(2) << " # First forecast loop with stochastic recruitment" << endl;
+  NuFore << Fcast_Loop_Control(3) << " # Forecast base recruitment:  0= spawn_recr; 1=mult*spawn_recr_fxn; 2=mult*VirginRecr; 3=deprecated; 4=mult*average_over_yr_range" << endl;
+  NuFore << "# for option 4, set phase for fore_recr_devs to -1 in control to get constant mean in MCMC, else devs will be applied" << endl;
```

Comment:
I would revise for clarity "for forecast base recruitment option 4 (mult*mean_over_yr_range),"